File: app/src/main/java/proficiency/android/com/data/model/api/ListDataResponse.java
package proficiency.android.com.data.model.api;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import proficiency.android.com.data.model.db.ListData;

public final class ListDataResponse {

    @Expose
    @SerializedName("title")
    private String title;

    @Expose
    @SerializedName("rows")
    private List<ListData> listData;

}

File: app/src/main/java/proficiency/android/com/data/model/api/ListDataResponse.kt
package proficiency.android.com.data.model.api

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import proficiency.android.com.data.model.db.ListData

class ListDataResponse {

    @Expose
    @SerializedName("title")
    val title: String? = null

    @Expose
    @SerializedName("rows")
    val listData: List<ListData>? = null
}
